composer.json

```diff
@@ -16,18 +16,19 @@
     ],
     "require": {
         "php": "^7.1",
+        "doctrine/phpcr-bundle": "^2.0@dev",
```

Comment:
As we have the decision `phpcr`or `orm` in configuration, this bundle shouldn't be required hard.

Reply:
Correct, it is only until the bundle is tagged, as the root project defines the stability of the project

Reply:
@dbu what do you think?

Reply:
i would do that in .travis.yml for ci, and if somebody wants to use this, they have to use dev minimum stabiltiy anyways to install the routing bundle.

Reply:
Also fine, just saw the Composer problems and thought I'd lend a hand … give me a few and I'll update :+1:

Reply:
BTW, happy to close this PR too … again, just trying to be helpful as with Silex going EoL, and Bolt moving to SF4, we're going to be seeing a lot more of each other :wink:

Reply:
i am glad if you can update to do the thing in .travis.yml , we appreciate you helping out. and very happy if this bundle becomes useful to bolt!